### PR TITLE
ICDS Dashboard: SDD detailing ui changes

### PR DESCRIPTION
**Pull request by saiprakash-v**
https://app.asana.com/0/1175081284767805/1175317382203698
##### SUMMARY

UI changes for SDD on web dashboard.
(This branch is created from https://github.com/dimagi/commcare-hq/pull/27522. So, merge this only after merging https://github.com/dimagi/commcare-hq/pull/27522)
(All the changes are available only to featured users)
##### FEATURE FLAG

ICDS: Enable access to pre-release features in the ICDS Dashboard reports

Screenshot from 2020-05-25 20-47-17
Screenshot from 2020-05-25 20-47-50
Screenshot from 2020-05-25 20-48-39
Screenshot from 2020-05-25 20-50-01
Screenshot from 2020-05-25 20-51-46

### CODE REVIEW

**Comment by saiprakash-v** [comment]
Dependency on backend changes in https://github.com/dimagi/commcare-hq/pull/27522. **Do not merge** this pr before merging that

**Comment by czue** [comment]
Let me know if you need me to review this. At first glance it looks more suitable for @Rohit25negi and other core dashboard devs.

**Review by Rohit25negi — CHANGES_REQUESTED** [review]
left a few clarification comments.

File: custom/icds_reports/static/js/directives/service-delivery-dashboard/service-delivery-dashboard.directive.js
Comment (Rohit25negi) [inline custom/icds_reports/static/js/directives/service-delivery-dashboard/service-delivery-dashboard.directive.js:90]: Should this be under FF?

**Comment by Rohit25negi** [comment]
could you also test it with FF off and ensure that the report is unchanged of your changes?

File: custom/icds_reports/static/js/directives/service-delivery-dashboard/service-delivery-dashboard.directive.js
Comment (saiprakash-v) [reply custom/icds_reports/static/js/directives/service-delivery-dashboard/service-delivery-dashboard.directive.js:421]: Sure

File: custom/icds_reports/static/js/directives/service-delivery-dashboard/service-delivery-dashboard.directive.js
Comment (saiprakash-v) [reply custom/icds_reports/static/js/directives/service-delivery-dashboard/service-delivery-dashboard.directive.js:90]: No. This will just connect the "See more" to the function defined in the controller.

**Comment by saiprakash-v** [comment]
> could you also test it with FF off and ensure that the report is unchanged of your changes?

Tested it. Everything looks good

**Comment by saiprakash-v** [comment]
@Rohit25negi I made the changes requested. Requesting for re-review

**Review by Rohit25negi — APPROVED** [review]

**Pull request merged**
